src/site/utilities/featureFlags.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs');
```

Comment:
I thought there was some value in keeping everything this file, so that if you need to use some of the data in here, you don't have to do require multiple files. That said, some of this might be better in the constants folder. Open to opinions there.

Reply:
I'm wondering what it would look like to apply the feature flag at the `module.exports` level instead of at the parent-level `require`. As in, if we could do something like - 


__facilities-fragments/healthCareRegionStaffBios.node.graphql.js__

```js
const someQuery = `
  { } 
`

module.exports = someQuery; 
applyFeatureFlag(module, featureGlags.FEATURE1);
```

Then maybe, a parent module can just require `facilities-fragments/healthCareRegionStaffBios.node.graphql.js` as usual and the feature-flagged version will be imported automatically for environments containing that feature. I definitely haven't totally thought this through so I could definitely be wrong, but just that it might be nice if feature-flag code could be abstracted a bit from other parent modules.

Reply:
I think that would work. I think I prefer that a little more. I'll take a crack at that and see how it goes.

Reply:
That approach seems simpler, so I've updated to it. I didn't make you specify a flag, since we can loop through all the flags pretty easily. One less thing to mess up. But it is an optional param you can pass.